### PR TITLE
chore: move cip-24 to review

### PR DESCRIPTION
**Pull request by ninabarbakadze**
## Overview

Moving cip-24 to status - Review

### CODE REVIEW

File: cips/cip-24.md
Comment (rootulp) [inline cips/cip-24.md:7]: The reference implementation section needs to be filled out.

**Review by jcstein — APPROVED** [review]

**Review by rootulp — APPROVED** [review]

**Review by ninabarbakadze — COMMENTED** [review]
let's merge? @jcstein

**Comment by jcstein** [comment]
yup was just waiting on another review 😅

**Pull request merged**
